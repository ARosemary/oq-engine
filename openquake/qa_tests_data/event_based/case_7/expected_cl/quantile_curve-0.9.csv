0.0 0.0,1.0000000E+00 9.9999999E-01 9.9999906E-01 9.9994390E-01 9.9859022E-01 9.8385648E-01 9.1299599E-01 7.4545219E-01 5.3917090E-01 3.4641389E-01 2.0346518E-01 1.3036024E-01 7.9053584E-02 4.2493098E-02 1.8948458E-02 6.7226949E-03 1.7510330E-03 2.7798285E-04 1.8590719E-05,1.0000000E+00 1.0000000E+00 1.0000000E+00 1.0000000E+00 9.9999929E-01 9.9995285E-01 9.9874351E-01 9.8576506E-01 9.2568738E-01 7.8516532E-01 5.7943161E-01 3.8615306E-01 2.4111286E-01 1.4657626E-01 9.1603026E-02 5.3094037E-02 2.6542246E-02 1.1067059E-02 3.5743616E-03
0.0 1.0,1.0000000E+00 9.9999982E-01 9.9999367E-01 9.9984895E-01 9.9770565E-01 9.8017912E-01 9.0738909E-01 7.4331006E-01 5.4560749E-01 3.5664896E-01 2.0347399E-01 1.0545807E-01 4.9195067E-02 2.0354282E-02 7.2142694E-03 2.1044357E-03 4.5577082E-04 7.9665603E-05 7.5022990E-06,1.0000000E+00 1.0000000E+00 1.0000000E+00 9.9999985E-01 9.9999444E-01 9.9985889E-01 9.9789643E-01 9.8264138E-01 9.2126506E-01 7.8361180E-01 5.8115923E-01 3.9545742E-01 2.5057587E-01 1.4119754E-01 6.9800071E-02 3.0486966E-02 1.1433774E-02 3.6354879E-03 8.9696092E-04
0.0 2.5,9.7787937E-01 9.2096250E-01 7.8717729E-01 5.6683183E-01 3.1635296E-01 1.3218439E-01 4.1885172E-02 9.8792361E-03 1.6542137E-03 1.7685679E-04 5.3774535E-06 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00,9.9880577E-01 9.9466907E-01 9.7637365E-01 9.1347508E-01 7.6046223E-01 5.1952241E-01 2.7779530E-01 1.1414207E-01 3.6559488E-02 9.1860157E-03 1.6028296E-03 1.7159554E-04 4.4013130E-06 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00 0.0000000E+00
