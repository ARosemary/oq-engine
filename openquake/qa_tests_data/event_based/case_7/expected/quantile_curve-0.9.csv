# source_model_tree_path=('b11',), gsim_tree_path=('CB',), investigation_time=50.0,imt=PGA
lon,lat,depth,poe-0.005,poe-0.007,poe-0.0098,poe-0.0137,poe-0.0192,poe-0.0269,poe-0.0376,poe-0.0527,poe-0.0738,poe-0.103,poe-0.145,poe-0.203,poe-0.284,poe-0.397,poe-0.556,poe-0.778,poe-1.09,poe-1.52,poe-2.13
0.00000,0.00000,0.00000,1.000000E+00,1.000000E+00,1.000000E+00,9.999958E-01,9.996293E-01,9.923649E-01,9.602442E-01,8.305166E-01,6.132590E-01,3.462302E-01,2.014838E-01,1.392920E-01,4.877058E-02,2.469009E-02,2.469009E-02,2.469009E-02,0.000000E+00,0.000000E+00,0.000000E+00
0.00000,1.00000,0.00000,1.000000E+00,1.000000E+00,9.999998E-01,9.999717E-01,9.987691E-01,9.812207E-01,8.997412E-01,7.407597E-01,4.908436E-01,3.296800E-01,1.392920E-01,7.225651E-02,4.877058E-02,4.877058E-02,2.469009E-02,0.000000E+00,0.000000E+00,0.000000E+00,0.000000E+00
0.00000,2.50000,0.00000,9.979706E-01,9.838365E-01,9.092820E-01,6.671289E-01,3.781149E-01,1.175031E-01,2.469009E-02,0.000000E+00,0.000000E+00,0.000000E+00,0.000000E+00,0.000000E+00,0.000000E+00,0.000000E+00,0.000000E+00,0.000000E+00,0.000000E+00,0.000000E+00,0.000000E+00
